Koans/AboutDelegates.cs

```diff
@@ -157,21 +157,21 @@ public void BuiltInActionDelegateTakesInt()
             //The first is Action<>. Action<> can take a variety of parameters and has a void return type.
             //  public delgate void Action<T>(T obj);
 
-            Action<int> i = AssertIntEqualsFourtyTwo;
+            Action<int> i = AssertIntEqualsFortyTwo;
             i((int)FILL_ME_IN);
         }
         [Step(13)]
         public void BuiltInActionDelegateTakesString()
         {
             // Because the delegate is a template, it also works with any other type. 
-            Action<string> s = AssertStringEqualsFourtyTwo;
+            Action<string> s = AssertStringEqualsFortyTwo;
             s((string)FILL_ME_IN);
         }
         [Step(14)]
         public void BuiltInActionDelegateIsOverloaded()
         {
-            //Action is an overloaded delegate so it can take more than one paramter
-            Action<int, string> a = AssertAddEqualsFourtyTwo;
+            //Action is an overloaded delegate so it can take more than one parametr
```

Comment:
Not a maintainer again, I think towards the end you may need to add either er or re so that it says paramet**er** or paramet**re**. Anyone depending on if you adhere to either the American or British English respectively.

Reply:
Thanks for the suggestion. Parameter is correct.
`parametre` is french (missing an accent) 🇫🇷 

> parametre
Misspelling of [parameter](https://en.wiktionary.org/wiki/parameter#English).

https://en.wiktionary.org/wiki/parametre